core/jazz_slack-event-handler/index.js

```diff
@@ -0,0 +1,311 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+"use strict";
+
+const configModule = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const utils = require("./components/utils.js")(); //Import the utils module.
+const errorHandlerModule = require("./components/error-handler.js");
+const failureErrorCodes = require("./utils/failure-codes.js");
+const customErrorHandlerModule = require("./components/custom-error-handler.js"); //Import the custom error codes module.
+const format = require("string-template");
```

Comment:
I didn't see this being used anywhere, can you check and remove if not required?